### ISSUE TITLE
Is there a way to pipe errors from sentry to kibana/logstash?

### ISSUE THREAD

**Issue by phun-ky** [getsentry/sentry#2216]
I've only found a output plugin for logstash to sentry, but we would like sentry to pipe errors to logstash.

This is because frontend monitoring is just a fraction of the system we currently develop, and the system administrators would like to only have one view of the state of the system, and that's logstash (dashboard).

Is this possible?

**Comment by dcramer** [comment]
There might be a third party integration but we don't bake this kind of support into core. 

Also of note sentry doesn't operate the same as logging as we sample data so it's not a 1:1 replacement. Youll also have complications piping to logstash as it doesn't fit our notification paradigm.

**Comment by phun-ky** [comment]
So, to get this straight, you have no support of merging feedback from sentry into a fullstack ops-center?

**Comment by dcramer** [comment]
We have no support for piping sentry events into logstash. We have no support (naively) for doing anything other than email and web hooks. We have a usable plugin API for things like notifications. 

It's important to not consider sentry logging. Would you send zendesk tickets and account leads into logstash? You might. 

We have no intention to pretend (or event realistically support the notion) that Sentry is just a UI on logs. The data is drastically different and heavier and thus is treated differently.

tl:dr you could likely do this with the plugin API but we're never going to build it into sentrys core

**Comment by phun-ky** [comment]
I agree with your standing here, and respect it, I just wanted to make life easier for our administrators. We currently have 2 large screens with elastic/kibana/logstash and build server/load. This leaves not much room for any frontend-specific reporting.

I will try to find a solution to this.

**Comment by dcramer** [comment]
The main concerns with sentry forwarding data (assuming it's all events) would be how the data gets to the receiving end. Locally you usually use UDP which is fairly safe, but over the Internet that's much more complex (and you also need to deal with auth). I'm happy to provide direction on how it could be done but we're not logstash users ourselves so I can't really answer how you'd resolve the receiving end of things.

**Comment by phun-ky** [comment]
I think we could solve this with webhooks, will try to remember to post here with the solution for future reference. Thanks for the time, you have a great product!

**Comment by dcramer** [comment]
@phun-ky you can use webhooks but the caveat is by default we dont send a notification for all events

You would be able to create a rule that says "on every event send this webhook", but its likely the world will explode on a situation where you have a huge failure rate

**Comment by wildloop** [comment]
https://github.com/javiermatos/logstash-output-sentry

**Comment by almogyehuda** [comment]
> I think we could solve this with webhooks, will try to remember to post here with the solution for future reference. Thanks for the time, you have a great product!

@phun-ky Got any solution for this? Thanks :)

**Comment by phun-ky** [comment]
@almogyehuda sorry no, not long after my post I had to switch projects :(
